code/modules/atmospherics/machinery/datum_pipeline.dm

```diff
@@ -149,6 +149,8 @@
 /datum/pipeline/proc/merge(datum/pipeline/parent_pipeline)
 	if(parent_pipeline == src)
 		return
+	if(!parent_pipeline)
```

Comment:
Why are you adding this again? This got through in the forge PR. Also doing `if(!parent_pipeline || parent_pipeline == src)` would've been a lot better.

Reply:
I forgot to remove it. I added it because it was annoying me!